global/scripts/CorrThick.sh

```diff
@@ -0,0 +1,136 @@
+pipedirguessed=0
+if [[ ""$HCPPIPEDIR:-"" == "" ]]
+then
+	pipedirguessed=1
+	#fix this if the script is more than one level below HCPPIPEDIR
+	export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+#description of script/command
+opts_SetScriptDescription "Run curvature-corrected cortical thickness python script and save curvatures, regression coefficients, and curvature-corrected thickness"
+
+opts_AddMandatory '--subject-dir' 'SubjectDir' 'path' "folder containing all subjects"
+opts_AddMandatory '--subject' 'Subject' 'subject ID' "subject-id"
+opts_AddOptional '--regnamestr' 'RegNamesStr' 'my reg str' "set the desired registration name(s) separated by @, 'string' 'RegName@RegName@RegName@...etc.' default MSMSulc" "MSMSulc"
+opts_AddOptional '--hemi' 'Hemi' 'hemisphere' "provide hemisphere for regression calculation, L=Left, R=Right, or B=Both, default 'B'" "B"
+opts_AddOptional '--surf' 'Surface' 'surface' "provide surface for regression calculation, white or midthickness, default midthickness" "midthickness"
+opts_AddOptional '--patch-size' 'PatchSize' 'distance' "provide patch kernel size in millimeters FWHM for regression, default 6" "6"
+opts_AddOptional '--surf-smooth' 'SurfSmooth' 'distance' "provide surface smoothing in millimeters FWHM, default 2.14" "2.14"
+opts_AddOptional '--metric-smooth' 'MetricSmooth' 'distance' "provide metric smoothing in millimeters FWHM, default 2.52" "2.52"
+opts_AddOptional '--resample-only' 'ResampleOnly' 'YES or NO' "whether or not to generate the curvature-corrected cortical thickness data, if it is already available, but just to resample it to 164k and 32k, defaults to 'NO'" "NO"
+
+opts_ParseArguments "$@"
+
+if ((pipedirguessed))
+then
+	log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+#display the parsed/default values
+opts_ShowValues
+
+#set paths
+NonlinearFolder="$SubjectDir"/"$Subject"/MNINonLinear
+NativeFolder="$NonlinearFolder"/Native
+T1wFolder="$SubjectDir"/"$Subject"/T1w/Native
+
+#Make intermediate directory to save intermediate files
+mkdir -p "$NativeFolder"/CorrThick
+
+#Loop through left and right hemispheres
+if [[ "$Hemi" == *B* ]]
+then
+	Hemi="L R"
+fi
+
+RegNames=`echo "$RegNamesStr" | sed s/"@"/" "/g`
+
+LowResMesh="32"
+HighResMesh="164"
+MapListFunc="MRcorrThickness MRcorrThickness_intercept MRcorrThickness_normcoeffs MRcorrThickness_curvs MRcorrThickness_coeffs"
+
+#Generate MRcorrThickness in Native Space
+if [[ "$ResampleOnly" == "NO" ]]; then
+	for Hemisphere in $Hemi ; do
+		if [[ "$Hemisphere" == "L" ]] ; then
+			Structure="CORTEX_LEFT"
+		elif [[ "$Hemisphere" == "R" ]] ; then
+			Structure="CORTEX_RIGHT"
+		fi
+			(
+				cd "$HCPPIPEDIR"/global/scripts/
+				python CorrThick.py "$SubjectDir" "$Subject" "$Structure" "$Hemisphere" "$Surface" "$PatchSize" "$SurfSmooth" "$MetricSmooth"
+			)
```

Comment:
I would leave the `(` and `)` on the same level as the preceeding `fi` (so that the interior is only one level more indended than the `fi`).